models/runtime/index.js

```diff
@@ -10,20 +10,25 @@ module.exports = () => {
     namespace: 'runtime',
 
     state: {
-      code: ''
+      code: {}
```

Comment:
changes runtime to handle multiple codes selectable by groupIds, namely: 1, 2, 3, 4, ....